README.md

```diff
@@ -1,8 +1,70 @@
 # pulp-2to3-migration-test-fixtures
 
-This repository contains a mongodump archives, /var/lib/pulp/content archives, and /var/lib/published archives.
+This repository contains Pulp 2 snapshots in the form of mongodump archives, /var/lib/pulp/content
+ archives, and /var/lib/pulp/published archives to used in tests for the pulp-2to3-migration plugin.
+ 
 
-The following commands were used to generated the current snapshots. 
+### Structure
+
+There is a separate directory for each Pulp 2 snapshot.
+Each directory has to have:
+ - a mongodump
+ - a /var/lib/pulp/content snapshot
+ - a /var/lib/pulp/published snapshot
+ 
+Each directory should have a config to generate the Pulp 2 snapshot.
+It helps to regenerate it if needed and it serves as a documentation of what is present in the
```

Comment:
s/It/Config helps

Reply:
s/it/to regenerate the snapshot

Reply:
you are too specific ;), the word `it` is perfect, it covers anything one needs :P